src/qbo-optimization/qbo-pnl-report-snaps.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+import gluestick as gs
+import os
+import json
+
+from datetime import date
+from dateutil.relativedelta import relativedelta
+
+# Get the sync_type (full_sync or incremental_sync)
+sync_type = os.environ.get("SYNC_TYPE", "incremental_sync")
+
+INPUT_DIR = "./sync-output"
+SNAPSHOT_DIR = "./snapshots"
+OUTPUT_DIR = "./etl-output"
+
+# By default hotglue gets the last 2 periods + the current period
+LOOKBACK_PERIODS = 2
+
+# Get the report periods from the config file
+with open("config.json", "r") as f:
+    config = json.load(f)
+    LOOKBACK_PERIODS = config.get("report_periods", 2)
```

Comment:
I think we need to `- 1` from here for it to work properly if I'm not mistaken, as it includes the current period